### PR TITLE
Add test 'CRUD search'

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Add test 'CRUD search'
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | TEST_PATH="functional/BO/13_shopParameters/07_search/01_search/01_CRUDSearch.js" URL_FO=yourShopURL npm run specific-test

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/21869)

### CODE REVIEW

File: tests/UI/campaigns/functional/BO/13_shopParameters/07_search/01_search/01_CRUDSearch.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+// Import expect from chai
+const {expect} = require('chai');
+
+// Browser and tab
+let browserContext;
+let page;
+let numberOfSearch = 0;
+
+const createAliasData = new SearchFaker();
+const editSearchData = new SearchFaker({alias: createAliasData.alias});
```
Comment (boubkerbribri) [inline tests/UI/campaigns/functional/BO/13_shopParameters/07_search/01_search/01_CRUDSearch.js:31]: Any reason why we don't update the alias ?

**Review by boubkerbribri — APPROVED** [review]

File: tests/UI/campaigns/functional/BO/13_shopParameters/07_search/01_search/01_CRUDSearch.js (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/UI/campaigns/functional/BO/13_shopParameters/07_search/01_search/01_CRUDSearch.js:31]: When we update the alias, a new alias is created.
I don't know, maybe it's an expected behavior, we should verify it

Comment (boubkerbribri) [reply tests/UI/campaigns/functional/BO/13_shopParameters/07_search/01_search/01_CRUDSearch.js:31]: this could be an issue

Comment (nesrineabdmouleh) [reply tests/UI/campaigns/functional/BO/13_shopParameters/07_search/01_search/01_CRUDSearch.js:31]: yes, it's a bug, I will create an issue :+1:

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
@SimonGrn OK for merging ?

**Pull request merged**